scripts/benchmark/resvg.js

```diff
@@ -6,13 +6,9 @@ export default svg => (
 			loadSystemFonts: false,
 		},
 		logLevel: 'off',
-		imageRendering: 1,
-		textRendering: 0,
-		shapeRendering: 0,
 		fitTo: {
 			mode: 'width',
 			value: 800,
 		},
-		dpi: 2400,
```

Comment:
How can I make resvg-js set the output DPI to 2400?

Seems adding this back doesn't really make a difference. The resulting DPI is still 72.

#### resvg-js
![Screen Shot 2022-08-11 at 1 09 55 AM](https://user-images.githubusercontent.com/1075694/184068219-2e1f3786-50b1-4291-a121-e2fd15ebe6f9.png)

#### sharp
![Screen Shot 2022-08-11 at 1 10 16 AM](https://user-images.githubusercontent.com/1075694/184068218-6f222844-4c0f-48b7-9653-8b8cd5a8f5d9.png)

Reply:
See: https://github.com/RazrFalcon/resvg/issues/526#issuecomment-1190433890

Reply:
And https://github.com/RazrFalcon/resvg/issues/451#issuecomment-914478375